internal/checker/checker.go

```diff
@@ -516,7 +516,7 @@ type Program interface {
 	GetResolvedModule(currentSourceFile *ast.SourceFile, moduleReference string) *ast.SourceFile
 }
 
-type Host interface{}
+type Host any
```

Comment:
This is intentional; the host will eventually contain methods.

Reply:
OK, I set it back to empty interface.